inc/sitemaps/class-sitemaps-admin.php

```diff
@@ -159,7 +142,7 @@ public function status_transition_bulk_finished() {
 			return;
 		}
 
-		$options = WPSEO_Options::get_option( 'wpseo_xml' );
+		$options = WPSEO_Options::get_option( 'wpseo' );
```

Comment:
I became a big fan of wrapping dependencies, because this makes it easier to mock without modifying the option data.